src/Panel/Panel.php

```diff
@@ -45,7 +200,290 @@ public static function customCss(App $kirby)
     }
 
     /**
-     * Returns predefined icons path as sprite svg file
+     * Check for a custom js file from the
+     * config (panel.js)
+     *
+     * @param \Kirby\Cms\App $kirby
+     * @return string|false
+     */
+    public static function customJs(App $kirby)
+    {
+        if ($js = $kirby->option('panel.js')) {
+            $asset = asset($js);
+
+            if ($asset->exists() === true) {
+                return $asset->url() . '?' . $asset->modified();
+            }
+        }
+
+        return false;
```

Comment:
While we have the chance to change the return signature, I think we should change this and the `customCss()` method to return `null` in this case. Then we can type-hint the return value as `?string`.

Reply:
Sounds good 👍